### ISSUE TITLE
Blobs as outputs

### ISSUE THREAD

**Issue by VladStojDev** [ashawkey/stable-dreamfusion#58]
Hi all,

I keep getting just basic blobs/clouds of smoke as the output when I try and run some custom prompts. This is the prompt/config call I am using:

 python3 main.py --text "a green monster truck" --workspace trial -O --guidance stable-diffusion --seed 12387651
 
 I also tried without any seed parameter. Any ideas?
 
 Thanks in advance!

**Comment by mfogelson** [comment]
I think it is likely just the number of training steps. 

The default is only set to 10000, which ends up being a total of like 100 epochs. 

Try something like --iters 100,000 (Will take much longer, but should be better)

Other option is to look into finding better pre-training weights. Likely you have no checkpoints downloaded from the model and so it is training from scratch, this will also impede performance.

**Comment by VladS-PCH** [comment]
Thanks, I will try - Though even with my dual RTX A6000's, with 100 epochs it takes like 5 minutes...
Will update you once I run some more experiments! Thanks for you suggestion!
